core/src/main/java/me/lihq/game/people/Player.java

```diff
@@ -227,9 +227,11 @@ private void checkForExtraScore() {
             score += extra;
             if (extra == 0) {
                  game.screenManager.navigationScreen.speechboxMngr.addSpeechBox( new SpeechBox("There appear to be no extra points here, the cash pile before you is fake!"));
+                game.currentSnapshot.player.canMove = false;
             } else {
                 SpeechBox speechBox = new SpeechBox("You gained " + extra + " extra points! Lucky you :D");
                 speechBox.setTrigger(true);
+                game.currentSnapshot.player.canMove = false;
```

Comment:
Should this line ` game.currentSnapshot.player.canMove = false;` be within both cases of the if statement? Could probs move it out of the if brackets as that code is run every time, and doesn't depend on extra==0